embark-consult.el

```diff
@@ -225,13 +225,18 @@ count of the matches (there may be more than one match per line).
 The function FOOTER is called to insert a footer."
   (let ((buf (generate-new-buffer "*Embark Export Grep*")))
     (with-current-buffer buf
-      (insert (propertize header 'wgrep-header t 'front-sticky t))
-      (let ((count (funcall insert lines)))
-        (funcall footer)
-        (goto-char (point-min))
-        (grep-mode)
-        (setq-local grep-num-matches-found count
-                    mode-line-process grep-mode-line-matches))
+      (grep-mode)
+      (setq-local mode-line-process grep-mode-line-matches)
+      (let ((inhibit-read-only t))
+        (insert (propertize header 'wgrep-header t 'front-sticky t))
+        (dlet ((compilation-filter-start (point)))
```

Comment:
Woah! I never knew dlet existed! But, why use it here at all? `compilation-filter-start` is a dynamic variable, isn't it? So that would make this do the same thing with let in place of dlet.